nexus_constructor/stream_fields_widget.py

```diff
@@ -417,7 +427,10 @@ def update_existing_stream_info(self, field):
         field = field.children[
             0
         ]  # only the first stream in the stream group can be edited currently
-        schema = field.writer_module
+        try:
+            schema = field.writer_module
+        except AttributeError:
+            pass
```

Comment:
warning print here?

Reply:
This was some debugging code that I forgot to remove. It has now been deleted.